Misc/NEWS.d/next/C API/2020-11-19-15-33-42.bpo-1635741.9tVsZt.rst

```diff
@@ -0,0 +1 @@
+Port _posixshmem extension module to multiphase initialization (:pep:`489`)
```

Comment:
nitpick: would you mind to add a trialing dot? :-)

```suggestion
Port _posixshmem extension module to multiphase initialization (:pep:`489`).
```

Reply:
Next time!